cilium/tox.ini

```diff
@@ -12,8 +12,6 @@ envdir =
     py27: {toxworkdir}/py27
     py38: {toxworkdir}/py38
 dd_check_style = true
-description =
-    py{27,38}: e2e ready
```

Comment:
Shouldn't we keep that so we can `ddev env test` locally, and have `skip_on_ci` skip the e2e test on CI?

Reply:
If add skip on ci on every e2e test then the step fails on CI because it does not find any results